src/App.tsx

```diff
@@ -462,6 +463,9 @@ function AppContent() {
                 <Route exact path={APP_PATHS.PROJECT_VIEW}>
                   <ProjectView />
                 </Route>
+                <Route exact path={APP_PATHS.PROJECT_EDIT}>
+                  <ProjectEditView />
+                </Route>
```

Comment:
Not directly related to this PR,  thoughts on delegating routes downstream?
App.tsx would ideally have something like
```
{featureFlagProjects && (
  <Route path={APP_PATHS.PROJECTS}>
    <projects root component />
  </Route>
)}
```
Projects/index.tsx would have the individual routes. This helps keep App.tsx simpler and higher level.
Tried this approach for planting sites and it seemed cleaner to me.

Reply:
Great idea!